### ISSUE TITLE
mysterious green figure

### ISSUE THREAD

**Issue by SaucyX** [ThankTheSergal/Kase#6]
So this table seems really polished and contains a lot of useful modding tools, however, using this table has made this mysterious green guy show up at random times and the sky turns black. Is there a way to stop him from turning me upside down and giving me souls? I'd really like to continue using this table for my modding experience. And don't worry, no online for me except when I want to "shadowrealm" my friends!

**Comment by ThankTheSergal** [comment]
No, you see this is like the perk from Fallout. Comes with the table.
